rails-commenting.rb

```diff
@@ -5,27 +5,29 @@
 
 # app/controller/blog_posts_controller.rb
 
-# 1)
+# $ rails g controller blog_posts
+
+# 1) After we generate a controller, a class is created inside the controller file for Blog Posts. The controller class inherits from ApplicationController. 
 class BlogPostsController < ApplicationController
   def index
-    # 2)
+    # 2) Defined a method called index that creates an instance variable of @posts that gets all of the Blog Posts that have been created in the database.
     @posts = BlogPost.all
   end
 
   def show
-    # 3)
+    # 3) Defined a show method that creates an instance  variable of @blog, which will show a specific blog post when searched for in the URL with its assigned id. Example: localhost:3000/blog_posts/3
```

Comment:
Absolutely, I think I would also want to mention a bit more about the params portion though.